src/csharp/Grpc.Core.Api/AsyncUnaryCall.cs

```diff
@@ -93,13 +93,27 @@ public Task<Metadata> ResponseHeadersAsync
         }
 
         /// <summary>
-        /// Allows awaiting this object directly.
+        /// Gets an awaiter used to await this <see cref="AsyncUnaryCall{TResponse}"/>.
         /// </summary>
+        /// <returns>An awaiter instance.</returns>
+        /// <remarks>This method is intended for compiler user rather than use directly in code.</remarks>
```

Comment:
nit: "for compiler user" is a typo?

Reply:
I copied this comment from dotnet/runtime.

TBH I'm not sure. "usage" seems more correct.